autopush-common/src/db/dynamodb/mod.rs

```diff
@@ -543,6 +551,9 @@ impl DbClient for DdbClientImpl {
                 retryable_delete_error(self.metrics.clone()),
             )
             .await?;
+        self.metrics
+            .incr_with_tags("notification.message.removed")
```

Comment:
We call it `deleted` in the other impl, unless you want to rename it:
```suggestion
            .incr_with_tags("notification.message.deleted")
```

Reply:
updated the spreadsheet to include `notification.message.deleted` 
thanks!